src/core/src/op/fake_convert.cpp

```diff
@@ -40,6 +40,35 @@ const std::string& FakeConvert::get_destination_type() const {
 void FakeConvert::validate_and_infer_types() {
     OV_OP_SCOPE(v13_FakeConvert_validate_and_infer_types);
     validate_type();
+    for (size_t i = 0; i < get_input_size(); i++) {
+        switch (get_input_element_type(i)) {
+        case element::bf16:
+        case element::f16:
+        case element::f32:
+        case element::dynamic:
+            break;
+        default:
+            OPENVINO_THROW("The element type of the input tensor on index ",
+                           i,
+                           " must be a bf16, f16, f32 or dynamic (got ",
+                           get_input_element_type(i),
+                           ").");
+        }
+    }
+    if (inputs().size() == 3) {
+        OPENVINO_ASSERT(get_input_partial_shape(1).compatible(get_input_partial_shape(2)),
+                        "FakeConvert scale shape: ",
+                        get_input_partial_shape(1),
+                        " is not compatible with shift shape: ",
+                        get_input_partial_shape(2));
+    }
+    auto data_pshape = get_input_partial_shape(0);
+    NODE_VALIDATION_CHECK(
+        this,
+        PartialShape::broadcast_merge_into(data_pshape, get_input_partial_shape(1), op::AutoBroadcastType::NUMPY),
```

Comment:
The logic of `broadcast_merge_into` seems to be a little bit more complicated than needed for validation puprpose only. But it's acceptable as unified way of doing broadcast in shape_infer.

Reply:
Yeah, I agree that this might not be most optimal solution, but it was simplest to implement unidirectional broadcast check. It could be optimized in separate PR